src/skmultiflow/meta/adaptive_random_forest_regressor.py

```diff
@@ -0,0 +1,497 @@
+                 warning_detection_method: BaseDriftDetector=ADWIN(0.01),
+                 # Tree parameters
+                 max_byte_size=33554432,
+                 memory_estimate_period=1000000,
+                 grace_period=200,
+                 split_confidence=0.0000001,
+                 tie_threshold=0.05,
+                 binary_split=False,
+                 stop_mem_management=False,
+                 remove_poor_atts=False,
+                 no_preprune=False,
+                 leaf_prediction='perceptron',
+                 nb_threshold=0,
+                 nominal_attributes=None,
+                 learning_ratio_perceptron=0.02,
+                 learning_ratio_decay=0.001,
+                 learning_ratio_const=True,
+                 random_state=None):
+        """AdaptiveRandomForestRegressor class constructor."""
+        super().__init__()
+        self.n_estimators = n_estimators
+        self.max_features = max_features
+        self.disable_weighted_vote = disable_weighted_vote
+        self.lambda_value = lambda_value
+        if isinstance(drift_detection_method, BaseDriftDetector):
+            self.drift_detection_method = drift_detection_method
+        else:
+            self.drift_detection_method = None
+        if isinstance(warning_detection_method, BaseDriftDetector):
+            self.warning_detection_method = warning_detection_method
+        else:
+            self.warning_detection_method = None
+        self.instances_seen = 0
+        self._train_weight_seen_by_model = 0.0
+        self.ensemble = None
+        self.random_state = random_state
+        # This is the actual random_state object used
+        self._random_state = check_random_state(self.random_state)
+
+        # Regression Hoeffding Tree configuration
+        self.max_byte_size = max_byte_size
+        self.memory_estimate_period = memory_estimate_period
+        self.grace_period = grace_period
+        self.split_confidence = split_confidence
+        self.tie_threshold = tie_threshold
+        self.binary_split = binary_split
+        self.stop_mem_management = stop_mem_management
+        self.remove_poor_atts = remove_poor_atts
+        self.no_preprune = no_preprune
+        self.leaf_prediction = leaf_prediction
+        self.nb_threshold = nb_threshold
+        self.nominal_attributes = nominal_attributes
+        self.learning_ratio_perceptron = learning_ratio_perceptron
+        self.learning_ratio_decay = learning_ratio_decay
+        self.learning_ratio_const = learning_ratio_const
+        self.random_state = random_state
+
+    def partial_fit(self, X, y, sample_weight=None):
+        """ Partially (incrementally) fit the model.
+
+        Parameters
+        ----------
+        X : numpy.ndarray of shape (n_samples, n_features)
+            The features to train the model.
+
+        y: numpy.ndarray of shape (n_samples)
+            An array-like with the class labels of all samples in X.
```

Comment:
This should be updated to the regression case.

Reply:
Done, could you check & resolve?